qemu/qemu.go

```diff
@@ -1689,3 +1725,14 @@ func LaunchCustomQemu(ctx context.Context, path string, params []string, fds []*
 	}
 	return errStr, err
 }
+
+func init() {
+	if runtime.GOARCH == "s390x" {
+		VhostUserNet = "virtio-net"
+		VhostUserSCSI = "vhost-user-scsi"
+		VirtioSerial = "virtio-serial"
+		VirtioNetPCI = "virtio-net"
+		Virtio9P = "virtio-9p"
+
+	}
+}
```

Comment:
I'd explicitly use `-ccw`

Reply:
Maybe also a short comment that on `s390x` the `-pci` devices don't properly work. That's why you introduce the init function